src/DayColumn.js

```diff
@@ -13,6 +13,18 @@ import * as DayEventLayout from './utils/DayEventLayout'
 import TimeSlotGroup from './TimeSlotGroup'
 import TimeGridEvent from './TimeGridEvent'
 
+function dfs(node, maxIdx, visited) {
```

Comment:
I put it outside on purpose bc of hoisting. and also I think it's better than declared inside of class.